doc/UsingUT.md

```diff
@@ -0,0 +1,99 @@
+<!--
```

Comment:
Thanks ever so much for updating the documentation...

Did you edit and create mdsource/.source.md files - if so, please could you add them to the PR?

If not, then they are explained here: https://github.com/approvals/ApprovalTests.cpp/blob/master/doc/Contributing.md#documentation

Note that I've just pushed some updates to master, to the generated documentation, with the latest mdsnippets tool - apologies if this creates conflicts for you.

Let me know if you have any questions about this... if it saves you time, it's fine for you to leave the docs in these normally-generated files for now, and I can move the text around before merging...

Reply:
Hi, the only new file I created for the documentation is this doc/UsingUT.md, which I copy/pasted and then changed from the doc/UsingDoctest.md. I didn't see the doc\mdsource subdirectory. I just added the file now.

Reply:
Thanks for this... We will need to get your other markdown edits in the master source.md files, so that they don't get overwritten next time the docs are generated... If you'd like me to do this after the merge, then please resolve this conversation....